examples/mouse_position.rs

```diff
@@ -60,8 +60,7 @@ fn update_cursor_state_from_window(
 
             if let Some(val) = window.cursor_position() {
                 action_state
-                    .action_data_mut(&driver.action)
-                    .unwrap()
+                    .action_data_mut_or_default(&driver.action)
```

Comment:
We should make `action_data` and `action_data_mut` infallible again, and remove this. Seperate PR though; I don't care whether it's before or after this PR is merged.